vllm/model_executor/layers/mamba/mamba_mixer2.py

```diff
@@ -209,8 +211,8 @@ def loader(param: torch.Tensor, loaded_weight: torch.Tensor) -> None:
 
 
 # Adapted from transformers.models.mamba.modeling_mamba.MambaMixer
-@CustomOp.register("mamba_mixer2")
-class MambaMixer2(MambaBase, CustomOp):
```

Comment:
main issue that needs solving: i guess this will cause V0 to break, need to find a way to make it work with both.